server/app/views/applicant/ApplicantProgramInfoView.java

```diff
@@ -74,7 +74,9 @@ private DivTag topContent(String programTitle, String programInfo, Messages mess
                 BaseStyles.TEXT_SEATTLE_BLUE, "text-2xl", "font-semibold", "text-gray-700", "mt-4");
 
     // "Markdown" the program description.
-    ImmutableList<DomContent> items = TextFormatter.formatText(programInfo, false);
+    ImmutableList<DomContent> items =
+        TextFormatter.formatText(
```

Comment:
This is a program details page that I think might be deprecated. It's accessible by going to `/applicants/:applicantId/programs/:programId`, but I don't think you can get here by clicking anywhere on the home page. I'm updating it for now, but I think we might be repurposing this page in the future. See more discussion in [this comment thread](https://docs.google.com/document/d/1NvtSTFWYG3Q5sVBZHNsjWLbd8X9BC5baVYA37fqzMS0/edit?disco=AAAA7CfyKII)